GoRogue/MapGeneration/PolygonArea.cs

```diff
@@ -155,14 +317,120 @@ private IEnumerable<IReadOnlyArea> GetBoundariesContaining(int x, int y)
 
         /// <inheritdoc/>
         public bool Intersects(IReadOnlyArea area) => _points.Intersects(area);
+        #endregion
 
-        /// <inheritdoc/>
-        public Rectangle Bounds => _points.Bounds;
+        #region Transformation
 
-        /// <inheritdoc/>
-        public int Count => _points.Count;
+        /// <summary>
+        /// Moves the Polygon in the indicated direction.
+        /// </summary>
+        /// <param name="dx">The X-value by which to shift</param>
+        /// <param name="dy">The Y-value by which to shift</param>
+        /// <returns></returns>
```

Comment:
General comment pertaining to the documentation of _all_ of the translate functions:

> Moves the Polygon in the indicated direction.
This combined with nothing being listed in the `<returns>` tag could create some confusion.  I would suggest wording the summary in a way that indicates that it returns a _copy_ of the polygon that has its points moved in the given direction (as applicable for each transformation function).